.github/workflows/tcoffee.yml

```diff
@@ -0,0 +1,30 @@
+name: FastQC
```

Comment:
The name is probably left over from the fastqc template that this was based on

Reply:
Solved in 787bf44c62c42019585a1a112644203fb21c08ef